docs/api/api-getting-started.md

```diff
@@ -0,0 +1,241 @@
+	- The `client_id` referenced in the protocol documentation for both the `sub` and `iss` values should be the `client_id` issued to the client upon registration.
+	- The `aud` value that is expected in incoming JWT assertions is the Sara Alert token endpoint.
+		- Development: `http://localhost:3000/oauth/token`
+		- Demo: `https://demo.saraalert.org/oauth/token`
+		- Production: `https://sara.public.saraalert.org/oauth/token`
+    - The `jti` value should be a string value that uniquely identifies the JWT among requests from the client application. Therefore how it is generated is up to the client application. It's even okay to make it as simple as a counter, but the client application must ensure it will never be used twice on two different JWTs. 
+    - The `exp` value must be epoch time in *seconds* and should be no more than 5 minutes in the future. 
+2. Request a new access token via HTTP POST to the FHIR authorization server’s token endpoint URL which is again `<ENVIRONMENT_BASE_URL>/oauth/token`
+3. Once the end-user has authorized the request, Sara Alert will respond with an access token.
+
+##### POST `/oauth/token`
+
+**Request** Headers
+
+    Content-Type: application/x-www-form-urlencoded
+
+**Request** Body
+
+    client_assertion: <CLIENT_SIGNED_JWT_ASSERTION>,
+    client_assertion_type: urn:ietf:params:oauth:client-assertion-type:jwt-bearer,
+    grant_type: client_credentials,
+    scope: <CLIENT_SCOPES (space separated)>
+
+**Response**
+
+    access_token: <TOKEN>,
+    token_type: "Bearer",
+    expires_in: 7200,
+    scope: <CLIENT_SCOPES>,
+    created_at: 1589830122
+
+#### Authentication
+The obtained token can then be used for all subsequent requests, via Authorization header, i.e. `'Authorization': "Bearer <TOKEN>"` and will be used to authenticate the user.
+
+Note that access tokens expire after two hours, and after a given access token expires the client application must go through the authorization process once more. Because the authorization process is automated - this workflow can still be fully automated.
+
+Example request using access token:
+
+##### GET `[base]/Patient/[:id]`
+
+**Request Headers**
+
+```
+  Content-Type: application/x-www-form-urlencoded,
+  Accept: application/fhir+json,
+  Authorization": Bearer <ACCESS_TOKEN>
+```
+
+#### Testing
+Sara Alert has a script that generates demo data for testing provided in the source code [here](https://github.com/SaraAlert/SaraAlert/blob/master/lib/tasks/demo.rake). This demo data includes a read/write OAuth 2.0 application for testing this workflow.
+
+Developers can use the same syntax to create a test application with a `jurisdiction_id`, a `user_id` and a `public_key_set`.
+NOTE: The public_key_set *must* be serialized into a JSON string using `to_json` before storing.
+
+Fortunately, there are many tools available and is easy to generate JWKS at https://mkjwk.org/ for the public key set, which also then be used to create a JWT at https://jwt.io.
```

Comment:
https://myjwk.org and https://jwt.io aren't showing up as clickable links on the generated page. See image below
![image](https://user-images.githubusercontent.com/52747882/102381368-1c291c80-3f97-11eb-87d6-266897dd6943.png)

Reply:
Fixed https://github.com/SaraAlert/SaraAlert/pull/598/commits/490bef24961bf80cb728c31cd3185f0d063a4d0b